website/blog/2023-3-28-aws-storage-class.md

```diff
@@ -0,0 +1,134 @@
+---
+slug: aws-storage-class
+title: "Choosing the Best AWS S3 Storage Class for Corso Backups"
+description: "By watching my
+four short videos, you’ll be set up and on your way to backing up all your Microsoft 365
+data in less than 20 minutes. That’s shorter than most meetings!"
+authors: nica
+tags: [corso, microsoft 365, AWS, backups]
+date: 2023-3-28
+image: ./images/box_sizes.jpeg
+---
+
+![multiple box sizes](./images/box_sizes.jpeg)
+Recently when writing
+about the storage options for Corso, I found myself going pretty far in the
+weeds on storage classes in S3. I thought I’d make a list of all the storage
+options and why they might, or might not, work for backups.
+<!-- truncate -->
+
+## First, some assumptions
+
+If we’re talking about backups, we're assuming far more writes than read, and
+that most objects that are written will never be read.
+
+Increasing age of an object increases the chances that it will never be read.
+
+And we can’t afford to lose anything! This isn't a store for ML inputs or other
+data where can afford to lose a percentage. One-zone options that carry a small
```

Comment:
It is questionable if the ML example is appropriate here. While some intermediate ML data may be recreated, good training data is an incredibly valuable asset. I would just remove this sentence.